recipe/migrations/cuda111_112.yaml

```diff
@@ -29,6 +29,11 @@ __migrator:
       - quay.io/condaforge/linux-anvil-cuda:11.0    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cuda:11.1    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cuda:11.2    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+
+      - quay.io/condaforge/linux-anvil-cos7-cuda:9.2     # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+      - quay.io/condaforge/linux-anvil-cos7-cuda:10.0    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+      - quay.io/condaforge/linux-anvil-cos7-cuda:10.1    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+      - quay.io/condaforge/linux-anvil-cos7-cuda:10.2    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
```

Comment:
This is the only change needed. Please remove other changes

Reply:
No, please test it locally. If other changes are removed I got the same errors when rerendering.

Reply:
What you are proposing will add 9.2, 10.0, 10.1 to the build matrix and we've decided not to add them by default.

Reply:
It's not up to me, it's up conda-smithy. I am not proposing anything, just wanna remove the error and unblock the migrator.

Reply:
Please propose a PR that will not expand the matrix to 9.2, 10.0, 10.1

Reply:
But how do I verify it locally?